apps/docs/src/guide/addresses/conversion.md

```diff
@@ -0,0 +1,41 @@
+# Converting Between Address Formats and Native Types
+
+This guide demonstrates how to convert between address formats and Sway Standard Types using helper functions. Native types are wrappers for bytes, and you can perform conversions between them by leveraging these functions and classes.
+
+## From `Bech32` to `b256`
+
+By instantiating an `Address`, we can validate a `Bech32` address and easily convert it to a `b256`:
+
+<<< @/../../docs-snippets/src/guide/types/conversion.test.ts#conversion-5{ts:line-numbers}
+
+Or if you'd prefer to use utility functions directly for validation and conversion, you can use `isBech32` and `toB256`:
+
+<<< @/../../docs-snippets/src/guide/types/conversion.test.ts#conversion-6{ts:line-numbers}
+
+## From `b256` to `Bech32`
+
+In a similar fashion, we have both class functions on the `Address` and utilities available for `b256` validation and conversion:
+
+<<< @/../../docs-snippets/src/guide/types/conversion.test.ts#conversion-7{ts:line-numbers}
+
+And by using the `isB256` and `toBech32` utilities:
+
+<<< @/../../docs-snippets/src/guide/types/conversion.test.ts#conversion-8{ts:line-numbers}
+
+## Converting a Contract ID
+
+The Contract `id` property has the [`AbstractAddress`](../types/address#abstractaddress-class) type, therefore it be converted using the `Address` class functions such as `toAddress` and `toB256`;
```

Comment:
```suggestion
The Contract `id` property has the [`AbstractAddress`](../types/address#abstractaddress-class) type. Therefore, it can be converted using the `Address` class functions such as `toAddress` and `toB256`:
```